apps/src/templates/teacherDashboard/SectionTable.story.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import SectionTable from './SectionTable';
+
+const sections = [
+  {
+    id: 11,
+    name: "brent_section",
+    loginType: "word",
+    grade: null,
+    stageExtras: false,
+    pairingAllowed: true,
+    numStudents: 10,
+    code: "PMTKVH",
+    assignmentName: "CS Discoveries",
+    assignmentPath: "//localhost-studio.code.org:3000/courses/csd"
```

Comment:
I do think we try to, but I'm pretty sure storybook is used much more on localhost than elsewhere, and I'd rather it have the better experience.

For the most part, I would argue that it doesnt really make a difference, as I'm mostly not expecting links to work from storybook regardless.